spec/models/instrument_spec.rb

```diff
@@ -521,16 +521,55 @@
       assert @reservation1.valid?
       # find next reservation after 12 am tomorrow at 10 am tomorrow
       @next_reservation = @instrument.next_available_reservation(after: Time.zone.tomorrow.beginning_of_day)
-      assert_equal (Time.zone.now + 1.day).day, @next_reservation.reserve_start_at.day
+      assert_equal 1.day.from_now.day, @next_reservation.reserve_start_at.day
       assert_equal 10, @next_reservation.reserve_start_at.hour
     end
   end
 
+  context "next available reservation with daily booking" do
+    let!(:instrument) do
+      create(
+        :instrument,
+        facility:,
+        pricing_mode: Instrument::Pricing::SCHEDULE_DAILY,
```

Comment:
Not a correction, but a question: why didn't you use the :daily_booking trait you've just created?

Reply:
good catch, I should have used it

Reply:
Great, same in line 986